lib/google/cloud/gemserver/cli/server.rb

```diff
@@ -170,6 +168,29 @@ def extract_key response
           end
 
           ##
+          # @private Creates a Gemfile and Gemfile.lock for the gemserver that
+          # runs on Google App Engine such that gemstash is not required
+          # client side for the CLI.
+          def gemfile
+            source = %{
```

Comment:
Consider a [squiggly heredoc](https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc) instead. That will make the indentation less ugly.